src/include/daos_srv/pool.h

```diff
@@ -92,8 +92,10 @@ struct ds_pool_child {
 	daos_handle_t	spc_hdl;
 	struct ds_pool	*spc_pool;
 	uuid_t		spc_uuid;
+	uint64_t	spc_rebuild_epoch;
```

Comment:
move to 'ds_pool'?

Reply:
this is per xstream thing.

Reply:
right, but all xstreams should share same value, right?

Reply:
not in this patch, but I will change it in the following patch to use unique global epoch.